site/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "devDependencies": {
+    "@symfony/webpack-encore": "^0.27.0",
+    "bootstrap": "^4.3.1",
+    "chartist": "^0.11.0",
+    "jquery": "^3.4.1",
+    "moment": "^2.24.0",
+    "node-sass": "^4.12.0",
+    "popper.js": "^1.15.0",
+    "sass-loader": "^7.1.0"
```

Comment:
All these dependencies are hard ones, not only development dependencies, as far as I can see.

Reply:
Is there some issue here?

Reply:
Yes, everything here should most likely not be a devDependency, but a normal dependency.